### PR TITLE
Adding Matrix and Genomeset types

### PR DESCRIPTION
**Pull request by slebras**

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -198,6 +234,41 @@ mappings:
...
+      row_ids: {type: keyword}
+      col_ids: {type: keyword}
+      num_rows: {type: integer}
+      num_columns: {type: integer}
+      attributes: {type: keyword}
+
+  "genomeset:1":
+    global_mappings: [ws_auth, ws_object]
+    properties:
+      genomes: {type: object}
```
Comment (jayrbolton) [inline config.yaml]: maybe you can add a nested type mapping for this object

File: config.yaml [hunk 2]
```diff
@@ -18,12 +25,35 @@ sdk_indexer_apps:
   KBaseGenomes.Genome:
```
Comment (jayrbolton) [inline config.yaml]: Remove?

Comment (slebras) [reply config.yaml]: removed

File: config.yaml (hunk 1 above)
Comment (slebras) [reply config.yaml]: 👍

**Comment by jayrbolton** [comment]
tudo bom

**Pull request merged**